lib/protocol/collection/item_collection.ts

```diff
@@ -117,23 +117,35 @@ export class ItemCollection extends MutableCollection<SNItem> {
       /** Filtered content type map */
       const filteredCTMap = this.filteredMap[contentType]!;
       const sortedElements = this.sortedMap[contentType]!;
+
+      const previousIndex = filteredCTMap[element.uuid];
+      const previousElement = !isNullOrUndefined(previousIndex) ? sortedElements[previousIndex] : undefined;
+      if (element.errorDecrypting || previousElement?.errorDecrypting) {
+        if (!element.errorDecrypting && previousElement?.errorDecrypting) {
+          /** If new element is not encrypted do a resort */
+         typesNeedingResort.add(contentType);
+        }
+         /** If both previous and new values of element are errored, we simply replace the value, as a re-sort
```

Comment:
This comment appears to no longer be valid since it's not addressing the case of both previous and new values being errored.